main.go

```diff
@@ -73,6 +99,28 @@ func main() {
 				"DO NOT USE in production",
 		)
 		vault, err = deterministic.NewStore(numDeterministicKeys)
+	case "mnemonic":
+		log.Warn(
+			"Using a mnemonic to recover keys from",
+		)
+
+		if mnemonicFile == "" {
+			log.Fatal("You must provide a --mnemonic-file")
+		}
+
+		content, err := ioutil.ReadFile(mnemonicFile)
+		if err != nil {
+			logrus.Info("File reading error", err)
```

Comment:
```suggestion
			log.Info("File reading error", err)
```

Reply:
Sure, don't know why I was using `logrus`. I might got confused by [this](https://github.com/prysmaticlabs/remote-signer/blob/master/main.go#L100). Should this also be `log` instead of `logrus`?

Reply:
Yes, you can correct it in this PR if you want to